### PR TITLE
Feature: frejya, user feedback

### PR DESCRIPTION
**Pull request by ele828**

### CODE REVIEW

**Comment by marcusrc** [comment]
Can one of the admins verify this patch?

File: test/messages/Messages.spec.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ describe('messages', () => {
     const domInput = searchInput.find('input').first();
     domInput.instance().value = 'something-doesnt-exist';
     domInput.simulate('change');
-    panel = wrapper.find(MessagesPanel).first();
+    panel = wrapper.find(MessageList).first();
```
Comment (ele828) [inline test/messages/Messages.spec.js]: this maybe wrong, double-check

File: dev-server/webpack.config.js [hunk 2]
```diff
@@ -101,6 +101,6 @@ const config = [{
   },
 }];
 
-config.port = '8080';
+config.port = '8191';
```
Comment (embbnux) [inline dev-server/webpack.config.js]: Why update this port?

File: src/components/FeedbackPanel/styles.scss [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+.instruction {
+  padding: 5px 0px 10px 0px;
+  @include primary-font;
+  font-size: 12px;
+}
+.explain {
+  font-weight: normal;
+}
+.select {
+  border: 1px solid #a6a6a6;
```
Comment (embbnux) [inline src/components/FeedbackPanel/styles.scss]: please use predefined color

**Comment by marcusrc** [comment]
Can one of the admins verify this patch?

**Pull request merged**